### PR TITLE
Update configuring android studio page

### PR DESCRIPTION
**Pull request by skyace65**
Updates the configuring android studio page with a new screenshot (went through png compression). minor update to the instructions. Also a minor fix, android studio isn't a JetBrains IDE, it's based on JetBrains Intellij IDE.

### CODE REVIEW

File: development/cpp/configuring_an_ide/android_studio.rst [hunk 1]
```diff
@@ -4,15 +4,14 @@ Android Studio
 ==============
 
 `Android Studio <https://developer.android.com/studio>`_ is a free
-`JetBrains <https://www.jetbrains.com/>`_ IDE for Android development.
+IDE for Android development based on `Intellij <https://www.jetbrains.com/>`__.
```
Comment (pycbouh) [inline development/cpp/configuring_an_ide/android_studio.rst:7]: Well, if we still link to the JetBrains website, it should also be okay to mention them by name. Android Studio *is* developer by both JetBrains and Google, so this statement wasn't wrong.

And if we were to mention the parent product, it's *IntelliJ IDEA*, not *Intellij*.

**Review by pycbouh — CHANGES_REQUESTED** [review]

**Comment by skyace65** [comment]
I tweaked the description, better?

File: development/cpp/configuring_an_ide/android_studio.rst [hunk 2]
```diff
@@ -4,15 +4,15 @@ Android Studio
 ==============
 
 `Android Studio <https://developer.android.com/studio>`_ is a free
-`JetBrains <https://www.jetbrains.com/>`_ IDE for Android development.
-It has a feature-rich editor which supports Java and C/C++. It can be used to
+IDE for Android development made by google and `JetBrains <https://www.jetbrains.com/>`_,
```
Comment (pycbouh) [inline development/cpp/configuring_an_ide/android_studio.rst:7]:
```suggestion
IDE for Android development made by `Google <https://about.google/>`_ and `JetBrains <https://www.jetbrains.com/>`_.
```

Proper capitalization and we can still link to Google (we do link to Microsoft in other articles).

File: development/cpp/configuring_an_ide/android_studio.rst [hunk 3]
```diff
@@ -4,15 +4,15 @@ Android Studio
...
+and based on `Intellij IDEA <https://www.jetbrains.com/idea/>`__. It has a
```
Comment (pycbouh) [inline development/cpp/configuring_an_ide/android_studio.rst:8]:
```suggestion
It's based on `IntelliJ IDEA <https://www.jetbrains.com/idea/>`__ and has a
```

**Review by pycbouh — CHANGES_REQUESTED** [review]
Yes, but still a couple of nitpicks.

**Comment by skyace65** [comment]
Fixed.

**Review by pycbouh — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you :)

**Pull request merged**